gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/WriteModuleXml.kt

```diff
@@ -220,12 +215,32 @@ open class WriteModuleXml @Inject constructor(_objects: ObjectFactory) : Default
         return false
     }
 
-    private fun manifests(): List<ArtifactManifest> {
-
-        return artifactManifests.get().map { manifest ->
+    // Manifests' artifacts, collect + sort by distinct scopes to de-dup them
+    private fun manifests(): List<Pair<String,String>> =
+        artifactManifests.get().map { manifest ->
             artifactManifestFromJson(manifest.asFile.readText(Charsets.UTF_8))
+        }.let { manifests ->
```

Comment:
Avoids having to build this collection twice, for second inner use on `#232 - #234`.